docs/concepts/results.md

```diff
@@ -1,7 +1,89 @@
 # TimeEval results
 
-How to work with the results of TimeEval.
+Upon executing TimeEval python script for the given datasets, timeeval checks the effectiveness of selected algorithm and generate an output file as `./TimeEval/results/<timestamp>` directory. The output directory has following data structure:
+
+```bash
+results/<time-stamp>/
+├── results.csv
+├── algorithm_1/hyperparameter/
+|   ├── dataset_1/epoch/
+│   |   ├── anomaly_scores.ts
+│   |   ├── docker-algorithm-scores.csv
+│   |   ├── execution.log
+│   |   ├── hyper_params.json
+│   |   └── metrics.csv
+|   └── dataset_2/epoch/
+│       ├── anomaly_scores.ts
+│       ├── docker-algorithm-scores.csv
+│       ├── execution.log
+|	├── model.pkl
```

Comment:
Please don't use tabs, always use spaces for indentation.

Reply:
i will keep in mind